mw-webapp/src/service/DayReportService.ts

```diff
@@ -1,19 +1,31 @@
-import {onValue, ref} from "firebase/database";
-import {DayReportDTOToDayReportConverter} from "src/converter/DayReportConverter";
+import {collection, getDocs} from "firebase/firestore";
+import {DayReportDTOToDayReportConverter, querySnapshotToDayReportDTOConverter} from "src/converter/dayReportConverter";
 import {db} from "src/firebase";
 import {DayReport} from "src/model/businessModel/DayReport";
 import {DayReport as DayReportDTO} from "src/model/firebaseCollection/DayReport";
+import {CurrentProblemService} from "src/service/CurrentProblemService";
+import {JobDoneService} from "src/service/JobDoneService";
+import {PathToCollection} from "src/service/PathToCollection";
+import {PlanForNextPeriodService} from "src/service/PlanForNextPeriodService";
 
 export class DayReportService {
 
-  public static onValueFromRealTimeDb(callBack: (data: DayReport[]) => void) {
-    onValue(ref(db, "/dayReports"), async (snapshot) => {
-      const dayReportsDTO: DayReportDTO[] = snapshot.val();
-      if (dayReportsDTO !== null) {
-        const dayReports = dayReportsDTO?.map((elem) => DayReportDTOToDayReportConverter(elem));
-        callBack(dayReports);
-      }
-    });
+  public static async getDayReports(): Promise<DayReport[]> {
+    const jobsDoneRaw = await JobDoneService.getJobsDone();
```

Comment:
service should not use other services.
aggregation is a business logic

Reply:
I don't understand then how to convert uuid strings to Ways if I dont get Ways

Reply:
use different models and layers

Reply:
Done